streams/src/test/java/org/apache/kafka/streams/kstream/internals/SuppressScenarioTest.java

```diff
@@ -529,6 +532,74 @@ public void shouldSupportFinalResultsForSessionWindows() {
         }
     }
 
+    @Test
+    public void shouldApplyNameToFinalSuppressionNode() {
```

Comment:
Just a couple of new tests to make sure that the node naming has the expected result on the topology.